README.md

```diff
@@ -45,18 +45,19 @@ In order for the generator to know what names to use for some parameters it's ne
-|disconnectionTimeout|Only for MQTT. The completion timeout in milliseconds when disconnecting. The default disconnect completion timeout is 5000 milliseconds.|No|`5000`|
-|completionTimeout|Only for MQTT. The completion timeout in milliseconds for operations. The default completion timeout is 30000 milliseconds.|No|`30000`|
-|asyncapiFileDir| Path where original AsyncAPI file will be stored.|No|`src/main/resources/api/`|
+|Name|Description|Required| Default                   |
+|---|---|---|---------------------------|
+|disableEqualsHashCode|Disable generation of equals and hashCode methods for model classes.|No| `false`                   |
+|inverseOperations|Generate an application that will publish messages to `publish` operation of channels and read messages from `subscribe` operation of channels. Literally this flag will simply swap `publish` and `subscribe` operations in the channels. <br> This flag will be useful when you want to generate a code of mock for your main application. Be aware, generation could be incomplete and manual changes will be required e.g. if bindings are defined only for case of main application.|No| `false`                   |
+|javaPackage|The Java package of the generated classes. Alternatively you can set the specification extension `info.x-java-package`. If both extension and parameter are used, parameter has more priority.|No| `com.asyncapi`            |
+|listenerPollTimeout|Only for Kafka. Timeout in ms to use when polling the consumer.|No| `3000`                    |
+|listenerConcurrency|Only for Kafka. Number of threads to run in the listener containers.|No| `3`                       |
+|addTypeInfoHeader|Only for Kafka. Add type information to message header.|No| `true`                    |
+|connectionTimeout|Only for MQTT. This value, measured in seconds, defines the maximum time interval the client will wait for the network connection to the MQTT server to be established. The default timeout is 30 seconds. A value of 0 disables timeout processing meaning the client will wait until the network connection is made successfully or fails.|No| `30`                      |
+|disconnectionTimeout|Only for MQTT. The completion timeout in milliseconds when disconnecting. The default disconnect completion timeout is 5000 milliseconds.|No| `5000`                    |
+|completionTimeout|Only for MQTT. The completion timeout in milliseconds for operations. The default completion timeout is 30000 milliseconds.|No| `30000`                   |
+|asyncapiFileDir| Path where original AsyncAPI file will be stored.|No| `src/main/resources/api/` |
+|mqttClientId| Only for MQTT. Provides the client identifier for the MQTT server.|No| guest                     |
```

Comment:
1. I suggest to move it one row higher. so, all "only mqtt" parameters will be together.
2. Are you sure that you need to change formatting of all rows?
3. I suggest to point priority of choosing parameter more clearly e.g. "Overrides the value of the client identifier for the MQTT server, if it is set in the AsyncAPI, otherwise it provides a default value"

Reply:
Hi @Tenischev 
1. Updated the text and position in `README.md` and `package.json`.
2. No it doesn't require formatting of all rows. In `if-elif-else` approach, the first check is to see if `params.mqttClientId` exists. When a default value is set for `mqttClientID` in `package.json` , this condition would always be true which is why I removed it and directly put the default value assignment in `application.yml`.

Reply:
@Tenischev Hi, please take a look at the updated changes.